Automattic/jetpack
# Boost: Add benefits interstitial

Fixes #23318

#### Changes proposed in this Pull Request:
- [x] Useed svelte-navigator package for routing
- [x] Added benefits page
- [x] Added benefits page content and style
- [ ] Activated button to purchase plan

#### Jetpack product discussion
pc9hqz-iD-p2

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
- Go to 'Boost dashboard'
- Add `#/upgrade` to the URL
- Check the design on different screen sizes. dVyv3AO1fXG0gprXywMuMh-fi-2130%3A7342

![image](https://user-images.githubusercontent.com/3737780/160062539-72b04ef7-0927-4f6d-83f3-e60fd02fb041.png)
